File: Assignment 4 - Lists in Java/Car.java

public class Car {

    private String makeAndModel;
    private int maximumNumberOfPassengers;
    private int numberOfDoors;

    Car(String makeAndModel, int maximumNumberOfPassengers, int numberOfDoors){
        this.makeAndModel = makeAndModel;
        this.maximumNumberOfPassengers = maximumNumberOfPassengers;
        this.numberOfDoors = numberOfDoors;
    }
    
    public String getMakeAndModel(){
        return this.makeAndModel;
    }
    public int getMaximumNumberOfPassengers(){
        return this.maximumNumberOfPassengers;
    }
    public int getNumberOfDoors(){
        return this.numberOfDoors;
    }
    public void setMakeAndModel(String m){
        this.makeAndModel = m;
    }
    public void setMaximumNumberOfPassengers(int m){
        this.maximumNumberOfPassengers = m;
    }
    public void setNumberOfDoors(int m){
        this.numberOfDoors = m;
    }
    public String toString(){
        return "Make and Model: " + this.makeAndModel + "\nMaximum Number of Passengers: " + this.maximumNumberOfPassengers + "\nNumber of Doors: " + this.numberOfDoors + "\n";
    }
}

File: Assignment 3 - Linked Lists/Car.py
class Car:
    def __init__(self, makeAndModel = "None assigned", numberOfDoors = 4, maximumNumberOfPassengers = 5):
        self.makeAndModel = makeAndModel
        self.numberOfDoors = numberOfDoors
        self.maximumNumberOfPassengers = maximumNumberOfPassengers

    def setMakeAndModel(self, mam):
        self.makeAndModel = mam

    def getMakeAndModel(self):
        return self.makeAndModel
    
    def getMaximumNumberOfPassengers(self):
        return self.maximumNumberOfPassengers
    
    def __str__(self) -> str:
        return  f"Make and Model:{self.makeAndModel}\nNumber of doors: {self.numberOfDoors}\nMaximum number of passengers: {self.maximumNumberOfPassengers}" 

if __name__ == "__main__":
    car1 = Car()
    car1.setMakeAndModel("Dodge Dart")
    print(car1.getMakeAndModel())
    print(car1.getMaximumNumberOfPassengers)
    print(car1)